### ISSUE TITLE
Invalid ClickHouse Queries

### ISSUE THREAD

**Issue by lucaderi** [ntop/ntopng#6422]
Due to the nature of ClickHouse **all** queries **must** have in the WHERE clause the time keys for which a table has been created. Example

```
CREATE TABLE IF NOT EXISTS `host_alerts` (
`rowid` UUID,
`alert_id` UInt32 NOT NULL,
`alert_status` UInt8 NOT NULL,
...
`user_label` String,
`user_label_tstamp` DateTime
) ENGINE = MergeTree() PARTITION BY toYYYYMMDD(tstamp) ORDER BY (tstamp);
```

must have in the WHERE clause **at least** tstamp, otherwise the query will result in a **linear scan**.

This is what happens for instance when you click on the info button of an alert of of a flow. This is the URL that is requested

http://XXXX:3000/lua/alert_details.lua?page=flow&status=historical&row_id=a5a4d78d-f747-4bbc-b188-1902cb2addf3

and having only row_id and no time index (will be first_seen for the flow_alerts, this is a linear scan that takes hours to complete.

In summary
- fix the above bug for flow_alerts
- review all other queries to make sure there are no other linear scans

**Comment by cardigliano** [comment]
Fixed in ca189ea4fd0b509b8237849909e34bf0e505f372 (alerts) and pro a43aa1a5ba7b40cf5a846fcb0b1a3435baf75f13 (historical flows)

**Comment by lucaderi** [comment]
This issue verified as fixed
